alpha/ui.cpp

```diff
@@ -156,10 +159,40 @@ void _TS_UI::task(void* parameter)
       TS_HAL.lcd_printf("%04d-%02d-%02d ", datetime.year, datetime.month, datetime.day);
       TS_HAL.lcd_printf("%02d:%02d:%02d\n", datetime.hour, datetime.minute, datetime.second);
       TS_HAL.lcd_printf("Battery: %d%%", TS_HAL.power_get_batt_level());
+	  
+	  //uploadflag should be set by serial, temporarily putting it in RADIO.CPP 
+	  //for testing , set uploadFlag to true 
+	  uploadFlag = TS_RADIO.getUploadFlag();
+	  uploadFlag = true;
+	  //if charging and connected to wifi, display upload messages.
       if (TS_HAL.power_is_charging())
       {
         TS_HAL.lcd_printf(", Charging");
+		if (wifiConnected && uploadFlag)
+		{
+			//attempt to upload 
+			//TS_HAL.fillScreen(0x0000);
+			//show uploading text in the middle tentatively
+			TS_HAL.lcd_cursor(50, 140);
+			//update on progress of upload
+			TS_HAL.lcd_printf("%s", uploadFlag ?"UPLOADING":"");
+			TS_HAL.lcd_printf("Connected to %s", TS_RADIO.getWiFiName());
+
+			//clear display
+		}
       }
+	  if (wifiConnected && uploadFlag)
```

Comment:
we probably should have a better code structure than a large ifelse

Reply:
will refactor this and clear it up, there are some debug duplicate lines in this section will remove them